src/server/routes.py

```diff
@@ -40,16 +41,19 @@ def table_of_contents(lang, year):
 @validate
 def contributors(lang, year):
     config = get_config(year)
+    ebook_size_in_mb = get_ebook_size_in_mb(lang, year)
     contributors_list = list(config["contributors"].items())
     random.shuffle(contributors_list)
     config["contributors"] = dict(contributors_list)
-    return render_template('%s/%s/contributors.html' % (lang, year), config=config)
+    return render_template('%s/%s/contributors.html' % (lang, year), config=config, ebook_size_in_mb=ebook_size_in_mb)
 
 
 @app.route('/<lang>/<year>/methodology')
 @validate
 def methodology(lang, year):
-    return render_template('%s/%s/methodology.html' % (lang, year))
+    config = get_config(year)
+    ebook_size_in_mb = get_ebook_size_in_mb(lang, year)
+    return render_template('%s/%s/methodology.html' % (lang, year), config=config, ebook_size_in_mb=ebook_size_in_mb)
 
 
 # Accessibility Statement needs special case handling for trailing slashes
```

Comment:
i fixed.

Reply:
It doesn't look like this was fixed @MSakamaki

Reply:
I'm Sorry, after merging into main, the table of contents was not displayed because the DEFAUL_YEAR was set to 2020.

It is now shown in the next fix.
https://github.com/HTTPArchive/almanac.httparchive.org/pull/1393/files#r521649741 

are you okay with this?
![2020-accessibility-statement](https://user-images.githubusercontent.com/2183053/98925918-f634d600-2519-11eb-8dda-7e490c690f2c.png)